scripts/start.py

```diff
@@ -139,6 +146,21 @@ def main(args=None):
             '--config', 'webpack.dev.config.ts', '--watch']))
         # Give webpack few seconds to do the initial compilation.
         time.sleep(10)
+        # Redis-cli is only required in a development environment.
+        python_utils.PRINT('Starting Redis development server.')
+        # Start the redis local development server. Redis doesn't run on
+        # Windows machine. Use daemonize argument to prevent redis-server from
+        # exiting on its own.
+        if not common.is_windows_os():
+            background_processes.append(
+                subprocess.call([
+                    './third_party/redis-cli-6.0.6/src/redis-server',
+                    (common.REDIS_CONF_PATH), '--daemonize', 'yes']))
+        else:
+            raise Exception(
+                'Redis command line interface is not installed because your ' +
+                'machine is on the Windows operating system. Caching will ' +
+                'not work on a non-Windows machine.')
```

Comment:
Since this is being repeated in two scripts, let us have a function in `common.py` to do this. Also `./third_party/redis-cli-6.0.6/src/redis-server` declare this as a constant.

Reply:
Done.